pkg/sql/pg_catalog_diff.go

```diff
@@ -185,6 +188,30 @@ func (p PGCatalogTables) rewriteDiffs(diffFile string) error {
 	return nil
 }
 
+// getNotImplementedTables retrieves tables that are not yet part of crdb
```

Comment:
Also in general please remember to add a period to comments (for correct grammar)

Reply:
Done.